Feature/logisland source of traffic

jerome73 opened the pull request:
This implements the source of traffic processor. 
The goal of such processor is to figure out the source of traffic for a given web session such as described in issue #348

mathieu-rossignol commented:
First quick and dirty review

- Should'nt we have web-analytics maven module (like for cyber security), as this processor seems very specific?
- Seems there is no documentation
- Should'nt we respect the naming convention : action-object? getSourceOfTraffic / addSourceOfTraffic?

jerome73 commented:
I created a module for logisland-web-analytics-plugin and placed the processor in it with the consolidateSession one
I added documentation to the processor
I renamed the processor in setSourceOfTraffic
